File: LV.1/나누어 떨어지는 숫자 배열/solution.py
def solution(arr, divisor):
    answer = []
    f = []
    for i in arr:
        if i % divisor == 0:
            answer.append(i)
        else:
            f.append(i)
    if len(f) == len(arr):
        return -1
    else:
        answer.sort()
        return answer

File: LV.1/나누어 떨어지는 숫자 배열/Solution.java
class Solution {
    public int[] solution(int[] arr, int divisor) {
        ArrayList<Integer> list1 = new ArrayList<Integer>();
        ArrayList<Integer> list2 = new ArrayList<Integer>();
        
        Arrays.sort(arr);
        
        for (int i=0; i < arr.length; i++) {
            if (arr[i] % divisor == 0) {
                list1.add(arr[i]);
            } else {
                list2.add(arr[i]);
            }
        }
        
        if (list2.size() == arr.length) {
            int[] answer = {-1};
            return answer;
        } 
        
        int[] answer = new int[list1.size()];
        for (int i=0; i<list1.size(); i++) {
            answer[i] = list1.get(i);
        }
        
        return answer;
    }
}
